skimage/measure/fit.py

```diff
@@ -610,19 +612,14 @@ def _dynamic_max_trials(n_inliers, n_samples, min_samples, probability):
     trials : int
         Number of trials.
     """
-    if n_inliers == 0:
-        return np.inf
-
-    if probability == 1:
-        return np.inf
-
-    if n_inliers == n_samples:
-        return 1
-
-    nom = math.log(1 - probability)
-    denom = math.log(1 - (n_inliers / n_samples) ** min_samples)
-
-    return int(np.ceil(nom / denom))
+    inlier_ratio = n_inliers / float(n_samples)
```

Comment:
```suggestion
    inlier_ratio = n_inliers / n_samples
```

This was a Python 2-ism. :joy: